qa/workunits/windows/test_rbd_wnbd.py

```diff
@@ -374,18 +374,33 @@ def cleanup(self):
             self.remove()
 
     @Tracer.trace
-    def init_fs(self):
-        if not self.mapped:
-            raise CephTestException("Unable to create fs, image not mapped.")
+    @retry_decorator()
+    def _init_disk(self):
+        cmd = ("powershell.exe", "-command",
+               f"Get-Disk -Number {self.disk_number} | "
+               "Initialize-Disk")
+        execute(*cmd)
 
+    @Tracer.trace
+    @retry_decorator()
+    def _create_partition(self):
         cmd = ("powershell.exe", "-command",
                f"Get-Disk -Number {self.disk_number} | "
-               "Initialize-Disk -PassThru | "
-               "New-Partition -AssignDriveLetter -UseMaximumSize | "
-               "Format-Volume -Force -Confirm:$false")
+               "New-Partition -AssignDriveLetter -UseMaximumSize")
         execute(*cmd)
 
-        # Retrieve the drive letter.
+    @Tracer.trace
+    @retry_decorator()
+    def _format_volume(self):
+        cmd = (
+            "powershell.exe", "-command",
+             f"(Get-Partition -DiskNumber {self.disk_number}"
+             "| ? { $_.DriveLetter }) | Format-Volume -Confirm:$false")
```

Comment:
Should there be a space here?
```suggestion
             " | ? { $_.DriveLetter }) | Format-Volume -Confirm:$false")
```

Reply:
And is `-Force` no longer needed?

Reply:
The whitespace is not mandatory but it could improve readability when the command is logged, so I'll add it.

About the ``-Force`` flag, WS 2022 seems to ignore it (it will gladly format existing volumes, with or without the force flag). If I remember correctly, that long command chain that is being replaced used to fail on WS 2016 without the ``-Force`` flag, so I guess it's worth keeping it after all.

Reply:
> If I remember correctly, that long command chain that is being replaced used to fail on WS 2016 without the `-Force` flag, so I guess it's worth keeping it after all.

Did you mean to add it back?  I only see the whitespace change.

Reply:
whops, missed it somehow. Thanks, updated.